.CI/create_feedstocks.py

```diff
@@ -254,10 +253,24 @@ def write_token(name, token):
                 team.url + "/memberships/" + new_member
             )
 
+    # Update status based on remote
+    subprocess.check_call(['git', 'checkout', os.environ.get('TRAVIS_BRANCH')])
+    subprocess.check_call(['git', 'pull', '--rebase', '--autostash'])
+
+    # Generate a fresh listing of recipes removed.
+    # This gets pretty ugly as we parse `git status --porcelain`.
+    #
+    # * Each line we get back is a change to a file in the recipe directory.
+    # * We narrow the list down to recipes that are staged for deletion (ignores examples).
+    # * Then we clean up the list so that it only has the recipe names.
+    removed_recipes = subprocess.check_output(['git', 'status', '--porcelain', recipe_directory_name])
```

Comment:
Also appears we need an extra argument here to make sure we get unicode on Python 3 :facepalm:.

xref: https://github.com/conda-forge/staged-recipes/pull/1470